astropy/tests/pytest_openfiles.py

```diff
@@ -41,10 +36,7 @@ def _get_open_file_list():
     files = []
     p = psutil.Process()
 
-    if importlib_machinery is not None:
-        suffixes = tuple(importlib_machinery.all_suffixes())
-    else:
-        suffixes = tuple(info[0] for info in imp.get_suffixes())
+    suffixes = tuple(importlib_machinery.all_suffixes())
```

Comment:
Same comment as above - this will create conflicts with https://github.com/astropy/astropy/pull/6606